UPGRADING.md

```diff
@@ -249,6 +249,20 @@ Care must be taken to ensure that the callback function provided is async-signal
 be executed in an asynchronous signal handler and so must not call any functions that are not Async-Signal-Safe.
 Further information can be found [here](https://wiki.sei.cmu.edu/confluence/display/c/SIG30-C.+Call+only+asynchronous-safe+functions+within+signal+handlers).
 
+### Metadata changes
+
+- Removed `packageName` from the `app` metadata tab, as the field is duplicated by `app.id`
+- Removed `versionName` from the `app` metadata tab, as the field is duplicated by `app.version` and this has been known to cause confusion amongst users in the past
+
+### Other changes
```

Comment:
This section can be removed from the upgrading guide as we've purposely tried to drive users to the online docs here, which will remain up-to-date: https://github.com/bugsnag/bugsnag-android/blob/ee529b788ab81ecb4a46f361e2453286c7980f1c/UPGRADING.md#error---event

Reply:
I see, makes sense - have removed 'other changes'